specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -12,6 +12,64 @@ using TypeSpec.Versioning;
 
 namespace Azure.AI.Projects;
 
+
+@doc("Labeled results per row.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v1)
+model LabeledRows {
+  @doc("The label associated with the evaluation.")
+  label: string;
+
+  @doc("Detailed reasoning or explanation for the evaluation.")
+  reasoning: string;
+
+  @doc("The version identifier for the evaluation model.")
+  version: string;
+}
+
+@doc("Summary across rows for evaluation.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v1)
+model MetricSummary extends Results {
+  type: "metricSummary";
+
+  @doc("Aggregated summary per evaluator across all rows.")
+    data: Record<{
+    score?: string;
+    gpt_score?: string;
+    threshold: string;
+    defect_rate?: string;
+    passing_rate?: string;
+    binary_aggregate: string;
+  }>;
+}
+
+@doc("Stored result details and storage location. Currently only blob storage support for evaluation result/result uploading.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v1)
+model Storage extends Results {
```

Comment:
Maybe we should call it "BlobStorageResults" instead of just "Storage"? When we are adding more storage types, this name could be confusing